File: task-manager-api/src/main/java/com/tm/dto/bean/task/TaskCompleteResponseDto.java
package com.tm.dto.bean.task;

import com.tm.dto.common.Errors;

public class TaskCompleteResponseDto {

    /**
     * タスクID
     */
    private String taskId;

    /**
     * Error結果
     */
    private Errors errors;

    public String getTaskId() {
        return taskId;
    }
    public void setTaskId(String taskId) {
        this.taskId = taskId;
    }
    public Errors getErrors() {
        return errors;
    }
    public void setErrors(Errors errors) {
        this.errors = errors;
    }

}

File: task-manager-app/src/app/dto/interface/task-complete-response.dto.ts
import { Errors } from "../common/errors";

/**
 * タスク完了レスポンスDTOクラス。
 */
export class TaskCompleteResponseDto {

    /**
     * タスクID
     */
    public taskId: string;

    /**
     * エラーコード
     */
    private errors: Errors[];

    public getTaskId(): string {
        return this.taskId;
    }
    public setTaskId(taskId: string): void {
        this.taskId = taskId;
    }
    public getErrors(): Errors[]|null|undefined {
        return this.errors;
    }
    public setErrors(errors: Errors[]) {
        this.errors = errors;
    }
    
}